src/software/simulated_tests/simulated_er_force_sim_test_fixture.cpp

```diff
@@ -1,5 +1,8 @@
 #include "software/simulated_tests/simulated_er_force_sim_test_fixture.h"
 
+// TODO (#2419): remove this
+#include <fenv.h>
```

Comment:
Is this being added to handle the SIGFPE errors we saw previously in the kickoff_friendly_er_force_test?

Reply:
yes, we disable sigfpe when the er force simulator is running